issue-006/solution.js

```diff
@@ -1,4 +1,15 @@
-// implement your solution here
-function selectSum() {
-
-}
\ No newline at end of file
+function selectSum(a) {
+    var sum = a.reduce(add, 0);
+    function add(a, b) {
+        if (typeof a !== "number") {
+            a = 0;
+            return a + b;
```

Comment:
`return a + b` is repeated in all the 3 branches of the _condition_, could you just put it outside the `if` blocks?